tests/SDAccel/README.md

```diff
@@ -0,0 +1,20 @@
+# SDAccel test cases
+
+## Set up
+* Requires "xilinx-xocc"
+* set XILINX_SDX for using xocc
```

Comment:
Most of the lines require some preformatted code, such as here
`XILINX_SDX` (i.e. add some back-ticks around)